filesystem/fs_interface.go

```diff
@@ -0,0 +1,14 @@
+package filesystem
+
+type Client interface {
+	Open() File
```

Comment:
I think we should also use the usual interface in io package.

On Sun, Mar 8, 2015 at 1:20 AM, Hongchao Deng <notifications@github.com>
wrote:

> In filesystem/fs_interface.go
> <https://github.com/taskgraph/taskgraph/pull/18#discussion_r25997619>:
>
> > @@ -0,0 +1,14 @@
> > +package filesystem
> > +
> > +type Client interface {
> > +	Open() File
>
> What about write() and sync()?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/taskgraph/taskgraph/pull/18/files#r25997619>.
>